bzt/modules/proxy2jmx.py

```diff
@@ -114,11 +113,75 @@ def startup(self):
                                   'DESKTOP_SESSION': None,
                                   'GNOME_DESKTOP_SESSION_ID': None,
                                   'KDE_FULL_SESSION': None})
+        elif is_windows():
+            self.log.info('Prepare chrome loader')
+            chrome_path = self._get_chrome_path()
+            if chrome_path:
+                self._prepare_chrome_loader()
+                new_path = join(self.engine.artifacts_dir, 'chrome-loader') + os.pathsep + os.getenv('PATH', '')
+                additional_env.update({
+                    'path_to_chrome': chrome_path,
+                    'additional_chrome_params': '--proxy-server="%s"' % self.proxy,
+                    'chrome_loader_log': join(self.engine.artifacts_dir, 'chrome-loader.log'),
+                    'path': new_path
+                })
+            else:
+                self.log.warning('Chrome not found')
```

Comment:
At my opinion warning is better because for the most part of users another browsers will be unworkable as we support chrome only. So I recognize chrome practically as requirement for proxy2jmx.